<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QWidget" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Node Manager</string>
  </property>
  <layout class="QVBoxLayout" name="main_lay">
   <item>
    <layout class="QGridLayout" name="paramaters_lay" columnstretch="0,1">
     <item row="0" column="0">
      <widget class="QLabel" name="context_lbl">
       <property name="text">
        <string>Context</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="context_cmb">
       <property name="maxVisibleItems">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="manager_tab">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="footer_lay">
     <item>
      <widget class="QProgressBar" name="main_prgbar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
